### ISSUE TITLE
update annotations 

### ISSUE THREAD

**Issue by ClusterWood** [mgmeyers/obsidian-zotero-integration#156]
This little template works fine as long as the file created does not exist. It overwrites/updates everything except the annotations when called a second time. After the second invocation, all the annotations are blank and will never be extracted again until I delete the file. When everything is updated why not the annotations?
What am I missing?

{% for a in annotations %}
> {{a.annotatedText}}
{% endfor %}

{% for b in notes %}
> {{b.note}}
{% endfor %}

{% for c in attachments %}
> {{c.itemKey}}
{% endfor %}

**Comment by 3outeille** [comment]
@ClusterWood Have you find a workaround ? Everything is erased when I import a second time

**Comment by ClusterWood** [comment]
No, unfortunately not. The only solution for me is to import annotations only after I have finished reading in Zotero.

**Comment by mgmeyers** [comment]
I've made some updates that may have fixed this issue. Try updating and see if you still have issues with this.

**Comment by Ea510chan** [comment]
> I've made some updates that may have fixed this issue. Try updating and see if you still have issues with this.

Hi mgmeyers, I still encounter this issue. Can you clarify the detail of updated version related to this issue?
